lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.module

```diff
@@ -255,24 +255,6 @@ function dosomething_campaign_run_get_parent_nid($nid) {
 }
 
 /**
- * Given a campaign form and form state, check to see if there is a campgin run.
- * Adds the campaign run nid to the campaign form.
- *
- * @param array $form
- *  A drupal form array of a campaign node
- * @param array $form_state
- *  A drupal form_state array of a campaign node
- */
-function dosomething_campaign_run_add_campaign_run_info(&$form, &$form_state) {
```

Comment:
from the mouth of @mikefantini "this is helpful because you can click to the current run node from the campaign edit form, maybe let's leave this, but move it near the dropdown